### PR TITLE
Tracebacks from endpoint failures are propagated to client

### PR DESCRIPTION
**Pull request by seanshi-scale**
Add traceback as a field on EndpointResponse for debugging purposes
Also fix unrelated table printing error
see sc-382133 and https://github.com/scaleapi/models/pull/3493

Example invocation:
`sync_endpoint.predict(EndpointRequest(**broken_args))`
which gives an EndpointResponse that looks like
```
status: FAILURE, result: None, result_url: None, traceback: Traceback (most recent call last):
  File "/app/hosted_model_inference/hosted_model_inference/inference/sync_inference/fastapi_server.py", line 92, in predict
    return run_predict(predict_fn, payload)
  File "/app/hosted_model_inference/hosted_model_inference/inference/common.py", line 195, in run_predict
    return predict_on_args(predict_fn, request_params.args, request_params.return_pickled)
  File "/app/hosted_model_inference/hosted_model_inference/inference/common.py", line 152, in predict_on_args
    output = predict_fn(**inputs)
TypeError: predict_fn() got an unexpected keyword argument 'x'

```

### CODE REVIEW

File: launch/client.py [hunk 1]
```diff
@@ -902,6 +902,7 @@ def _sync_request(
         resp = self.connection.post(
             payload=payload,
             route=f"{SYNC_TASK_PATH}/{endpoint_id}",
+            handle_bad_response=False,
```
Comment (seanshi-scale) [inline launch/client.py:905]: if we return http 200 + "failed" as part of the sync endpoint failing response, we can get rid of this "handle_bad_response" thing

File: launch/model_endpoint.py [hunk 2]
```diff
@@ -236,9 +242,10 @@ def predict(self, request: EndpointRequest) -> EndpointResponse:
         )
         return EndpointResponse(
             client=self.client,
-            status=TASK_SUCCESS_STATE,
+            status=raw_response.get("state"),
```
Comment (syandroo) [inline launch/model_endpoint.py:245]: should we consolidate on state vs status haha

**Review by syandroo — COMMENTED** [review]
don't forget to bump verison also

File: launch/model_endpoint.py (hunk 2 above)
Comment (seanshi-scale) [reply launch/model_endpoint.py:245]: I think we should tbh, I like status personally

**Pull request merged**